crates/oxc_language_server/src/main.rs

```diff
@@ -398,6 +402,45 @@ impl LanguageServer for Backend {
 
         Ok(None)
     }
+
+    async fn execute_command(&self, params: ExecuteCommandParams) -> Result<Option<Value>> {
+        if params.command == "oxc.applyAllFixesFile" && params.arguments.len() == 1 {
+            let text_document =
+                VersionedTextDocumentIdentifier::deserialize(params.arguments.first().unwrap())
+                    .unwrap();
+            let uri = text_document.uri;
+            if let Some(value) = self.diagnostics_report_map.get(&uri.to_string()) {
+                join_all(value.iter().filter(|report| report.fixed_content.is_some()).map(
+                    |report| async {
+                        let fixed_content = report.fixed_content.as_ref().unwrap();
+                        return self
+                            .client
+                            .apply_edit(WorkspaceEdit {
+                                #[expect(clippy::disallowed_types)]
+                                changes: Some(std::collections::HashMap::from([(
+                                    uri.clone(),
+                                    vec![TextEdit {
+                                        range: fixed_content.range,
+                                        new_text: fixed_content.code.clone(),
+                                    }],
+                                )])),
+                                ..WorkspaceEdit::default()
+                            })
+                            .await;
+                    },
+                ))
+                .await;
+
+                return Ok(None);
```

Comment:
There is still something wrong. I tested it with `pnpm run build && pnpm run install-extension`.
I need to close the file and reopen to see the changes (`onType`).
When I am closing it, VSCode is also question my if I want to override this file.

Reply:
I don't think this should be an issue anymore (I never had this problem, but who knows why).  Re-open this if you're still have issues with the latest changes.